jhipster-framework/src/main/java/tech/jhipster/config/JHipsterDefaults.java

```diff
@@ -126,7 +126,7 @@
     }
 
     interface Security {
-
+        String contentSecurityPolicy = "content-security-policy: \"default-src 'self'; frame-src 'self' data:; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://storage.googleapis.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self' data:";
```

Comment:
I'm not sure if this the correct format, I think should be only value

Reply:
pushed corection